app/src/main/java/com/example/ana/exampleapp/GPSManager.java

```diff
@@ -0,0 +1,190 @@
+package com.example.ana.exampleapp;
+
+import android.Manifest;
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.app.Service;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.location.Location;
+import android.location.LocationListener;
+import android.location.LocationManager;
+import android.os.Build;
+import android.os.Bundle;
+import android.os.IBinder;
+import android.provider.Settings;
+import android.support.v4.app.ActivityCompat;
+import android.util.Log;
+
+
+/**
+ * Created by Niels on 14.06.16.
```

Comment:
According to [Android Code Style Guidelines](http://source.android.com/source/code-style.html):

> Every class and nontrivial public method you write must contain a Javadoc comment with at least one sentence describing what the class or method does. This sentence should start with a third person descriptive verb.

You can add your name with `@author` as I have done in other classes.

In case you want to add your name in other classes you have modified (I think you should add it in the `TestActivity` and `ConfigurationActivity` classes, as you have added lot of code there), according to [Oracle Javadoc guide](http://www.oracle.com/technetwork/articles/java/index-137868.html): 

> Multiple @author tags should be listed in chronological order, with the creator of the class listed at the top.